app/plugin.go

```diff
@@ -197,7 +197,9 @@ func (a *App) InitPlugins(pluginDir, webappPluginDir string) {
 		a.SyncPluginsActiveState()
 		if pluginsEnvironment := a.GetPluginsEnvironment(); pluginsEnvironment != nil {
 			pluginsEnvironment.RunMultiPluginHook(func(hooks plugin.Hooks) bool {
-				hooks.OnConfigurationChange()
+				if err := hooks.OnConfigurationChange(); err != nil {
+					a.Log.Error(err.Error())
```

Comment:
Can we add some context to the log?
```suggestion
					a.Log.Error("Plugin OnConfigurationChange hook failed", mlog.Err(err))
```

Reply:
I can't test this offline right now. I've commited your suggestion and we'll see what the automated tests say.